### PR TITLE
fix: improve fuzzing

### PR DESCRIPTION
**Pull request by sakulstra**

### CODE REVIEW

File: ui/src/types.ts [hunk 1]
```diff
@@ -1,7 +1,11 @@
-export type Address = {
+export type SearchItem = {
```
Comment (sakulstra) [inline ui/src/types.ts:1]: renamed as i found it super confusing (given Address is 0xstring in viem)

File: ui/src/components/Search.tsx [hunk 2]
```diff
@@ -3,30 +3,30 @@
...
 import { Box } from './Box';
 import { SearchResult } from './SearchResult';
 
 const fuseOptions = {
   includeScore: true,
-  keys: ['searchPath', 'value'],
+  keys: ['searchPath', 'value', 'library', 'key'],
   threshold: 0.2,
-  ignoreLocation: true,
+  // ignoreLocation: false,
```
Comment (sakulstra) [inline ui/src/components/Search.tsx:15]: not ignoring location rates `lib.oracle` > `lib.asset.oracle` which is what we want.

File: ui/src/components/Search.tsx [hunk 3]
```diff
@@ -3,30 +3,30 @@
...
-import { type Address } from '@/types';
+import { type SearchItem } from '@/types';
 import Fuse, { FuseResult } from 'fuse.js';
 import { Box } from './Box';
 import { SearchResult } from './SearchResult';
 
 const fuseOptions = {
   includeScore: true,
-  keys: ['searchPath', 'value'],
+  keys: ['searchPath', 'value', 'library', 'key'],
```
Comment (sakulstra) [inline ui/src/components/Search.tsx:13]: adding the root (library) and the end (key) as dedicated results makes the behavior more natural as in this case locationSearch will no longer consider the distance (given `oracle` would exact match the key)

**Review by smbdy — APPROVED** [review]
LGTM!

**Pull request merged**